### PR TITLE
Arabic Translation

### PR DESCRIPTION
**Pull request by 3b7ameed**
GUI Translation To arabic

### CODE REVIEW

**Comment by dEBRUYNE-1** [comment]
Thanks for your (first) pull request! One remark, please squash your commits.

**Comment by 3b7ameed** [comment]
@dEBRUYNE-1  Hi mate , would you please point me how to do that ?

**Comment by NanoAkron** [comment]
Is there a more generic 'Arabic' flag, or is is OK to use the Egyptian one here?

**Comment by erciccione** [comment]
PR looks good, only need to squash as dEBRUYNE said, @3b7ameed i'm now on #monero-translations if you need help with that

**Comment by 3b7ameed** [comment]
@NanoAkron there's no "Arab" flag at the end you will have to use a country flag , and as of most of the translation contains traditional Arabic , in some cases i used Egyptian dialect, if any Arabian read the whole thing he will know it's translated by Egyptian

File: translations/monero-core_ar.ts [hunk 1]
```diff
@@ -1426,72 +1466,32 @@ Please upgrade or connect to another daemon</source>
...
-    <message>
-        <location filename="../pages/Transfer.qml" line="198"/>
-        <source>Medium (x20 fee)</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../pages/Transfer.qml" line="199"/>
-        <source>High (x166 fee)</source>
-        <translation type="unfinished"></translation>
+        <translation">هويه المعامله &lt;font size=&apos;2&apos;&gt;( Optional )&lt;/font&gt;</translation>
```
Comment (erciccione) [inline translations/monero-core_ar.ts]: that `"` need to be removed

**Pull request merged**